tests/src/test/scala/org/apache/openwhisk/core/scheduler/queue/test/MemoryQueueFlowTests.scala

```diff
@@ -1534,20 +1536,14 @@ class MemoryQueueFlowTests
           probe.expectTerminated(fsm, 10.seconds)
 
         case _ =>
-          // queue is stale and will be removed
+          fsm.underlyingActor.containers = Set(testContainerId)
           parent.expectMsg(staleQueueRemovedMsg)
+          parent.expectMsg(message)
+          // queue is stale and will be removed
           probe.expectMsg(Transition(fsm, state, Removing))
 
           fsm ! QueueRemovedCompleted
 
-          // queue should not be terminated as there is an activation
```

Comment:
why these lines are removed? since there is a container in https://github.com/apache/openwhisk/pull/5228/files#diff-4b5788222d7949baf4e09b570e4fa49d3fa79e5432ff50c798b04353758c8d4aR1539, I think message should not be sent to `QueueManager`?

Reply:
Oh, fixed.